test/integration/builder/updates.js

```diff
@@ -70,6 +70,80 @@ module.exports = function(knex) {
         });
     });
 
+    it('should immediately return updated value for other connections when updating row to DB returns', function() {
```

Comment:
This is the new test case... if we can setup knex oracledb dialect to work in a way that this tests passes, then probably most of the other failing oracle tests will start to work